### PR TITLE
Updated issue link

### PR DESCRIPTION
**Pull request by weitzhandler**
Many to many is hands down the top issue on the EF Core sorted by most commented and most reacted.
 - [Most commented](https://github.com/aspnet/EntityFrameworkCore/issues?q=is%3Aissue+is%3Aopen+sort%3Acomments-desc)
 - [Most upvotes](https://github.com/aspnet/EntityFrameworkCore/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)

### CODE REVIEW

File: entity-framework/core/what-is-new/ef-core-3.0/features.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ This characteristic makes them an excellent fit for database views in most scena
 Work on this feature has started but it isn't included in the current preview. 
 
 This feature is about enabling entities that store data in indexed properties instead of regular properties, and also about being able to use instances of the same .NET class (potentially something as simple as a `Dictionary<string, object>`) to represent different entity types in the same EF Core model.
-This feature is a stepping stone to support many-to-many relationships without a join entity, which is one of the most requested improvements for EF Core.
+This feature is a stepping stone to support many-to-many relationships without a join entity ([Issue #1368](https://github.com/aspnet/EntityFrameworkCore/issues/1368)), which is the most requested improvements for EF Core.
```
Comment (divega) [inline entity-framework/core/what-is-new/ef-core-3.0/features.md]: @weitzhandler Thanks for adding the link. It makes sense. 

Is this your way of voting for many-to-many here too? :smile: My originally intent was to convey that it is important, and honestly I don't think this page needs to say more about it.

In any case, there is a typo: improvements --> improvement

Comment (weitzhandler) [reply entity-framework/core/what-is-new/ef-core-3.0/features.md]: It IS the most commented and reacted issue, although personally to me is less important than say `JSON_VALUE`, TPT, Temporal Tables, anyway, let's leave the _one of_ in and just keep the link to the issue.
I'm not quite sure where the improvement section is.

Comment (divega) [reply entity-framework/core/what-is-new/ef-core-3.0/features.md]: @weitzhandler fine. I will change it as you suggest and merge. 

> I'm not quite sure where the improvement section is.

Oh, I was just trying to say that the word "improvements" plural needed to change to the singular form for the sentence to be correct, but never mind.

**Comment by divega** [comment]
Thanks @weitzhandler for your contribution.

**Pull request merged**